src/lib/util/Timestamp.js

```diff
@@ -153,114 +266,20 @@ class Timestamp {
-
-Timestamp.DDD =
-Timestamp.DDDD = time => {
-	const start = new Date(time.getFullYear(), 0, 0);
-	const diff = ((time.getMilliseconds() - start.getMilliseconds()) + (start.getTimezoneOffset() - time.getTimezoneOffset())) * MINUTE;
-	return String(Math.floor(diff / DAY));
-};
-
-Timestamp.d = time => {
-	const day = String(time.getDate());
-	if (day !== '11' && day.endsWith('1')) return `${day}st`;
-	if (day !== '12' && day.endsWith('2')) return `${day}nd`;
-	if (day !== '13' && day.endsWith('3')) return `${day}rd`;
-	return `${day}th`;
-};
-
-Timestamp.dd = time => DAYS[time.getDay()].slice(0, 2);
-
-Timestamp.ddd = time => DAYS[time.getDay()].slice(0, 3);
-
-Timestamp.dddd = time => DAYS[time.getDay()];
-
-Timestamp.X = time => String(time.valueOf() / SECOND);
-
-Timestamp.x = time => String(time.valueOf());
-
-// Times
-
-Timestamp.H = time => String(time.getHours());
-
-Timestamp.HH = time => String(time.getHours()).padStart(2, '0');
-
-Timestamp.h = time => String(time.getHours() % 12 || 12);
-
-Timestamp.hh = time => String(time.getHours() % 12 || 12).padStart(2, '0');
-
-Timestamp.a = time => time.getHours() < 12 ? 'am' : 'pm';
-
-Timestamp.A = time => time.getHours() < 12 ? 'AM' : 'PM';
-
-Timestamp.m = time => String(time.getMinutes());
-
-Timestamp.mm = time => String(time.getMinutes()).padStart(2, '0');
-
-Timestamp.s = time => String(time.getSeconds());
-
-Timestamp.ss = time => String(time.getSeconds()).padStart(2, '0');
-
-Timestamp.S = time => String(time.getMilliseconds());
-
-Timestamp.SS = time => String(time.getMilliseconds()).padStart(2, '0');
-
-Timestamp.SSS = time => String(time.getMilliseconds()).padStart(3, '0');
-
-// Locales
-
-/* eslint max-len:0 new-cap:0 */
-
-Timestamp.T = (time) => `${Timestamp.h(time)}:${Timestamp.mm(time)} ${Timestamp.A(time)}`;
-
-Timestamp.t = (time) => `${Timestamp.h(time)}:${Timestamp.mm(time)}:${Timestamp.ss(time)} ${Timestamp.A(time)}`;
-
-Timestamp.L = (time) => `${Timestamp.MM(time)}/${Timestamp.DD(time)}/${Timestamp.YYYY(time)}`;
-
-Timestamp.l = (time) => `${Timestamp.M(time)}/${Timestamp.DD(time)}/${Timestamp.YYYY(time)}`;
-
-Timestamp.LL = (time) => `${Timestamp.MMMM(time)} ${Timestamp.DD(time)}, ${Timestamp.YYYY(time)}`;
-
-Timestamp.ll = (time) => `${Timestamp.MMMM(time).slice(0, 3)} ${Timestamp.DD(time)}, ${Timestamp.YYYY(time)}`;
-
-Timestamp.LLL = (time) => `${Timestamp.LL(time)} ${Timestamp.T(time)}`;
-
-Timestamp.lll = (time) => `${Timestamp.ll(time)} ${Timestamp.T(time)}`;
-
-Timestamp.LLLL = (time) => `${Timestamp.dddd(time)}, ${Timestamp.LLL(time)}`;
-
-Timestamp.llll = (time) => `${Timestamp.ddd(time)} ${Timestamp.lll(time)}`;
-
-Timestamp.Z =
-Timestamp.ZZ = time => {
-	const offset = time.getTimezoneOffset();
-	return `${offset >= 0 ? '+' : '-'}${String(offset / -60).padStart(2, '0')}:${String(offset % 60).padStart(2, '0')}`;
-};
-
-/* eslint-enable id-length */
+/**
+ * The map tokens, mapped from the constants.
```

Comment:
This is inaccurate, see below

Reply:
Yeah my brain went missing when I wrote this line, I don't know what to write there 😂